modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/utils/RESTAPITestUtil.java

```diff
@@ -0,0 +1,377 @@
+/*
+*Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+
+package org.wso2.am.integration.tests.restapi.utils;
+
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.commons.lang.RandomStringUtils;
+import org.apache.cxf.jaxrs.impl.ResponseImpl;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import org.wso2.am.integration.test.utils.APIManagerIntegrationTestException;
+import org.wso2.am.integration.tests.restapi.RESTAPITestConstants;
+import org.wso2.carbon.automation.engine.exceptions.AutomationFrameworkException;
+import org.wso2.carbon.automation.test.utils.http.client.HttpRequestUtil;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.File;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * This class is used as a util class to test the REST API.
+ */
+public class RESTAPITestUtil {
+
+    private static final Log log = LogFactory.getLog(RESTAPITestUtil.class);
+    private DataDrivenTestUtils dataDrivenTestUtils = new DataDrivenTestUtils();
+
+    /**
+     * This method is used to test a given scenario using the REST API
+     *
+     * @param configFilePath location of the data file
+     * @param gatewayURL     gateway URL
+     * @param keyMangerURL   key manager URL
+     * @return true if all the scenarios in the current test is successful, otherwise false
+     */
+    public boolean testRestAPI(String configFilePath, String gatewayURL, String keyMangerURL) {
+
+        //this is the boolean which indicates the success or failure of the current test
```

Comment:
Use proper linguistics in writing comments. Make the first letter of the first word capital.